### PR TITLE
[fix] Use only one place to reject columns

### PR DESCRIPTION
**Pull request by hallelujah**
- Added a test
- Doing it in a single place so table columns are not retrieved again.

### CODE REVIEW

File: app/models/account.rb [hunk 1]
```diff
@@ -3,10 +3,15 @@
 require_dependency 'backend_client'
 
 class Account < ApplicationRecord
+  # Hack to remove payment_gateway_type, payment_gateway_options and deleted_at from @attributes
+  # It is enough for rails not persisting them in actual columns.
   def self.columns
-    super.reject { |c| c.name == "deleted_at" }
+    super.reject{|column| /\Apayment_gateway_(type|options)|deleted_at\Z/ =~ column.name }
```
Comment (Martouta) [inline app/models/account.rb]: I guess I am missing an important piece of information here but these 2 seem to be in the schema right now and I don't know if we have planned to remove them of why 🤔
https://github.com/3scale/porta/blob/d6a617f595fd519fc85b5d02ff94223d6a1b5445/db/schema.rb#L44-L45

Comment (hallelujah) [reply app/models/account.rb]: Sure we need to remove them one day

**Comment by guicassolato** [comment]
I think the reason we had in two places was separation of concerns, but it's an ~ugly hack~ safe measure anyway :)

File: app/models/account.rb (hunk 1 above)
Comment (Martouta) [reply app/models/account.rb]: I don't get why enforcing these 2 columns here but I see it is not newly introduced by you so I'll consider it out of scope 😄

**Comment by hallelujah** [comment]
> I think the reason we had in two places was separation of concerns, but it's an ugly hack safe measure anyway :)

I think I did :trollface: myself by putting it in a module. Since I guess I did it long ago.

**Pull request merged**